src/Microsoft.AspNetCore.Diagnostics.HealthChecks/HealthCheckResponseWriters.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNetCore.Diagnostics.HealthChecks
+{
+    public static class HealthCheckResponseWriters
+    {
+        public static Task WriteMinimalPlaintext(HttpContext httpContext, CompositeHealthCheckResult result)
+        {
+            if (httpContext == null)
+            {
+                throw new ArgumentNullException(nameof(httpContext));
+            }
+
+            if (result == null)
+            {
+                throw new ArgumentNullException(nameof(result));
+            }
+
+            httpContext.Response.ContentType = "text/plain";
+            return httpContext.Response.WriteAsync(result.Status.ToString());
+        }
+
+        public static Task WriteDetailedJson(HttpContext httpContext, CompositeHealthCheckResult result)
+        {
+            if (httpContext == null)
+            {
+                throw new ArgumentNullException(nameof(httpContext));
+            }
+
+            if (result == null)
+            {
+                throw new ArgumentNullException(nameof(result));
+            }
+
+            httpContext.Response.ContentType = "application/json";
+
+            var json = new JObject(
```

Comment:
Not for this PR, but maybe something we should consider as a format  - https://inadarei.github.io/rfc-healthcheck/#rfc.section.5

Reply:
It's been discussed, in general we're not that interested in defining a format for folks to use for health checks. We might even remove this code because I'm not sure we'd tell users to use it.